core/jobs_test.py

```diff
@@ -779,20 +779,21 @@ def test_validity_of_each_continuous_computation_class(self):
                         event_type),
                     event_services.BaseEventHandler))
 
-            rdc = klass._get_realtime_datastore_class()  # pylint: disable=protected-access
-            self.assertTrue(issubclass(
-                rdc, jobs.BaseRealtimeDatastoreClassForContinuousComputations))
```

Comment:
why was this removed?

Reply:
I've moved this to line 795 by mistake, I can change it back if you'd like